share/functions/fish_status_to_signal.fish

```diff
@@ -1,17 +1,10 @@
 function fish_status_to_signal --description "Print signal name from argument (\$status), or just argument"
+    __fish_make_completion_signals # Make sure signals are cached
     for arg in $argv
         if test $arg -gt 128
-            set -l signals SIGHUP SIGINT SIGQUIT SIGILL SIGTRAP SIGABRT SIGBUS \
-                SIGFPE SIGKILL SIGUSR1 SIGSEGV SIGUSR2 SIGPIPE SIGALRM \
-                SIGTERM SIGSTKFLT SIGCHLD SIGCONT SIGSTOP SIGTSTP \
-                SIGTTIN SIGTTOU SIGURG SIGXCPU SIGXFSZ SIGVTALRM \
-                SIGPROF SIGWINCH SIGIO SIGPWR SIGSYS
-            set -l sigix (math $arg - 128)
-            if test $sigix -le (count $signals)
-                echo $signals[$sigix]
-            else
-                echo $arg
-            end
+            # Important: each element of $__kill_signals is a string like "10 USR1"
+            string replace -r --filter "^$(math $arg-128) " SIG $__kill_signals |
```

Comment:
I think we should not use the "$()" feature just yet, to avoid prompt errors when upgrading